src/sim65/6502.c

```diff
@@ -215,7 +215,9 @@ int PrintCycles;
     ZPAddr = MemReadByte (Regs.PC+1);                           \
     Addr = MemReadZPWord (ZPAddr);                              \
     if (PAGE_CROSS (Addr, Regs.YR))                             \
+    {                                                           \
```

Comment:
> But that is what I mean: Where can I find this "style guide"

It's not complete but it's at https://github.com/cc65/wiki/wiki/C-and-Assembly-sources-style-guide

> or is there an automatic check where I can inspect the violations when I break it? (I'm familiar with "Resharper" in this respect)

No, there isn't. 

> (Not that I find the style very consistent nor I would prefer it... ;) )

As far as I know the C sources are VERY consistent (in contrast to the Asm sources). I personally don't like the style but I didn't introduce it.

> You making others really a hard time to contribute when even copy & paste of existing code leads to refinishing...

I'm sorry for that. Can you please point out where existing C code doesn't adhere to the things I mentioned?